### DISCUSSION TITLE
How to install xformer for setting up the app?

### DISCUSSION THREAD

**Discussion by echcon** [facebookresearch/xformers#1105] [General]
I am using Python 3.10, Torch 2.4, Apple Silicon chip M1Max.
This is the error log in the install time: 
`Collecting xformers
  Using cached xformers-0.0.28.post1.tar.gz (7.8 MB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/private/var/folders/d3/13vn5rt91cs5cvvp69fhydgh0000gn/T/pip-install-wfnbid1n/xformers_e68956655fcb4cbaa455617baab0bfc6/setup.py", line 24, in <module>
          import torch
      ModuleNotFoundError: No module named 'torch'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.`

**Comment by lw** [comment]
See this issue: https://github.com/facebookresearch/xformers/issues/740
